src/opentime/timeRange.h

```diff
@@ -82,6 +82,34 @@ class TimeRange {
         return _start_time < other.end_time_exclusive() && other._start_time < end_time_exclusive();
     }
 
+    bool before(TimeRange other) const {
```

Comment:
This is a question for the team at large. Is there a reason TimeRange uses pass-by-value instead of const references throughout this file?